src/data/mv_polynomial/basic.lean

```diff
@@ -728,9 +730,13 @@ finsupp.sum_zero_index
 section
 
 @[simp] lemma eval₂_add : (p + q).eval₂ f g = p.eval₂ f g + q.eval₂ f g :=
-finsupp.sum_add_index
-  (by simp [f.map_zero])
-  (by simp [add_mul, f.map_add])
+begin
+  classical,
+  exact
+  finsupp.sum_add_index
```

Comment:
Why not use standard formatting here?
```suggestion
  exact finsupp.sum_add_index
```

Reply:
I have no idea why I did this, I must not have been paying attention